man/z[.neptune.new.run.Run.Rd

```diff
@@ -0,0 +1,44 @@
+\name{[.neptune.new.run.Run}
```

Comment:
Let's base doc strings on this: https://docs.neptune.ai/api-reference/run#field-lookup